randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+            "On Normal difficulty, doing so without any other sources of damage requires 230 missiles."
+        ),
+    ],
+    "889553182": [
+        # Frozen Metroid Hop Tip
+        # Replaces Metroid scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Metroid",
+        (
+            "&main-color=#89D6FF;Jumping on the center of a frozen Metroid causes one to bounce on top of it.\n\n"
+            "When it breaks out of the ice, anyone on top is launched high in the air\n\nThis can be used to traverse "
+            "Metroid Quarantine A without Space Jump Boots, Morph Ball Bombs, or Spider Ball."
+        ),
+    ],
+    "4176439895": [
+        # Frozen Hunter Metroid Hop Tip
+        # Replaces Hunter Metroid scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Hunter Metroid",
+        (
+            "&main-color=#89D6FF;Jumping on the center of a frozen Hunter Metroid causes one to bounce on top of it."
+            "\n\nWhen it breaks out of the ice, anyone on top is launched high in the air.\n\n"
+            "This can be used to acquire Frost Cave's item without the Grapple Beam."
+        ),
+    ],
+    "425486343": [
+        # Arboretum door DBJ tip
+        # Replaces Arboretum big branch scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;It is possible to surmount the gate "
+            "at the top of this room with a carefully-timed double bomb jump."
+        ),
+    ],
+    "1102487982": [
+        # FCS climb tip
+        # Replaces Frigate Crash Site frigate scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;It is possible to climb up to this room's top door from the bottom. "
+            "There are multiple methods, the easiest of which only needs the "
+            "&main-color=#FF3333;Space Jump Boots&main-color=#89D6FF; to climb the crashed frigate's exterior."
+        ),
+    ],
+    "3737672425": [
+        # Monitor Station climb tip
+        # Replaces Monitor Station 2f left computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;By jumping along the rectangular extrusions on this building's exterior, "
+            "it is possible to reach the roof without Space Jump Boots."
+        ),
+    ],
+    "3893588753": [
+        # Monitor Station climb tip
+        # Replaces Monitor Station 2f middle computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;While on this building's roof, it is possible to reach Warrior Shrine without "
+            "lowering the bridge. To do so, stand on top of the roof computer's tallest monitor, "
+            "lock onto a Puffer to the right or left, and perform a lock-on dash towards the ledge."
```

Comment:
- Calling this a `lock-on dash` is a little confusing. I'd change it to `combat dash` to make it consistent with the trick name.
- It would also be worth mentioning that the gap can be traversed with an R-Jump.